.storybook/webpack.config.js

```diff
@@ -14,6 +14,9 @@ module.exports = ({config}) => {
 		test: /\.(ts|tsx)$/,
 		use: [require.resolve('awesome-typescript-loader')],
 	});
+
+	config.resolve.mainFields = ['ts:main', 'module', 'main'];
```

Comment:
Maybe in the place of `ts:main` would it be `source`? it may be clearer that it is not a file to be consumed by something from outside.

Reply:
The reason I went with `ts:main` is because I wanted it to be somewhat similar to `main` otherwise I thought someone might think its something entirely different.

Reply:
Oh yeah, makes sense. I do not see problem also in going with `ts:main`, just giving another name if `ts:main` is still doubtful.

Reply:
I still think their might be a better solution for this out there, but I just couldn't find anything for it. I'm sure someone has run into this issue before, I just couldn't find an example.

 Do you know of any lerna + ts libraries out there that are good examples?

@wincent any chance you've seen this before?

Reply:
Hm yeah, I did a little research but I also did not find anything related to this, it is quite difficult to see such setup with lerna+ts.

Reply:
> @wincent any chance you've seen this before?

I'm afraid not.